thirdparty/kubernetes-client-c/remove-findpackage.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/kubernetes/PreTarget.cmake b/kubernetes/PreTarget.cmake
+index 93ea00e..1e064b8 100644
+--- a/kubernetes/PreTarget.cmake
++++ b/kubernetes/PreTarget.cmake
+@@ -41,6 +41,3 @@ list(APPEND HDRS
+         websocket/kube_exec.h
+         include/generic.h
+         include/utils.h)
+-
+-find_package(libwebsockets CONFIG REQUIRED)
+-find_package(yaml CONFIG REQUIRED)
```

Comment:
Consider adding dummy find modules under `cmake/libwebsockets/dummy` and `cmake/yaml/dummy`. See the existing ones for example. These directories are added to the cmake module path in our root CMakeLists.txt. This approach would make find_package work.

Reply:
I think adding `Find*.cmake` files would only work for `find_package` in `MODULE` mode, and this is `CONFIG` mode, so CMake is looking for `yamlConfig.cmake` and `libwebsockets-config.cmake`.  These are, in fact, generated by CMake (at `build/_deps/yaml-build/CMakeFiles/yamlConfig.cmake` in the case of `libyaml`), but I haven't found a way of making CMake look for it there.  This issue: https://gitlab.kitware.com/cmake/cmake/-/issues/21687 may be about adding this capability to CMake, but it has not been resolved.

Reply:
According to the `find_package` docs, you could set `<PackageName>_ROOT` (`libwebsockets_ROOT` and `yaml_ROOT` in this case) to this directory and it would be found by `find_package`. The issue is about doing this automatically in cmake.

Reply:
Thanks, I did not know about the `*_ROOT` variable.  I have tried to make this work, but
 * I still need to patch the kubernetes lib to set the [CMP0074](https://cmake.org/cmake/help/latest/policy/CMP0074.html) policy to NEW, and
 * the generated `yamlConfig.cmake` and `libwebsockets-config.cmake` files don't quite work: `yamlConfig.cmake` refers to a `yamlTargets.cmake` file which does not exist, and `libwebsockets-config.cmake` tries to `include()` `LibwebsocketsTargets.cmake`, which does exist, but CMake says it "was generated by the export() command.  It may not be used as the argument to the include() command.  Use ALIAS targets instead to refer to targets by alternative names."

I'm sure all these problems could be solved, but at this point I think the least messy solution is to leave this as it is.